dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Semi-official Mercurial bridge from Git project"
+HOMEPAGE="https://github.com/mnauw/git-remote-hg"
+SRC_URI="https://github.com/mnauw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="
+	app-text/asciidoc
+"
+DEPEND="${PYTHON_DEPEND}"
+RDEPEND="
+	${DEPEND}
+	dev-vcs/git
+	dev-vcs/mercurial
+"
+
+# Some tests fail.
```

Comment:
Link upstream report.

Reply:
There is none. However, the test suite prints this:

`TODO known breakage`

For tests that fail. So upstream is aware.